ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -490,6 +493,13 @@ message DamlContractKeyState {
   google.protobuf.Timestamp active_at = 2;
 }
 
+message DamlContractKeyIdPair {
+  DamlContractKey key = 1;
+
+  // Contract id the key was resolved to.
+  string id = 2;
```

Comment:
Call this `resolved_to_contract_id` or `resolved_to_id` or similar so it's self-explanatory.

Reply:
That's a better name, thanks!